### PR TITLE
Added BGR2Gray

### PR DESCRIPTION
**Pull request by SimpleVlad**
- [ ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ ] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [ ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [ ] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+To pass the test on this challenge, change your <code>h1</code> element's text to say "Hello World".
+</section>
+
+## Tests
+<section id='tests'>
+
+```yml
+tests:
+  - text: Your <code>h1</code> element should have the text "Hello World".
+    testString: assert.isTrue((/hello(\s)+world/gi).test($('h1').text()), 'Your <code>h1</code> element should have the text "Hello World".');
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md]: It seems to me that this field contains JS code which tests user's code. Try to add `console.log(1)` here to see.

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      <canvas id="canvasOutput" ></canvas> 
+  </div>
+
+  <script async src="https://docs.opencv.org/master/opencv.js" onload="onOpenCvReady();" type="text/javascript"></script>
+  <script type="text/javascript">
+
+   function onOpenCvReady() {
+    document.getElementById('status').innerHTML = 'OpenCV.js is load.';
+
+    cv["onRuntimeInitialized"]=()=>{
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md]: I think we need to hide from user this initialization. Perhaps, it's better to replace OpenCV initialization to something like 
```html
   <script async src="https://docs.opencv.org/master/opencv.js"
                 onload="cv["onRuntimeInitialized"]=()=> { main() }"
                 type="text/javascript"></script>
```

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+<section id='challengeSeed'>
+
+<div id='html-seed'>
+
+```html
+ <h2>Gray && HSV filter in OpenCV.js</h2>
+  <p id="status">OpenCV.js is loading...</p>
+  <div>
+      <img id="imageSrc" src="https://bit.ly/fcc-relaxing-cat"/>
+      <div class="caption"> <input type="button" id="Run"  onclick="runDetectAndColor()" value="Run" /></div>
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md]: Please use lowercase word for `id`

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md (hunk 2 above)
Comment (dkurt) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md]: Oh, no. We need to enable Run button when OpenCV has been initialized.

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    
+    function onOpenCvReady() {
+      document.getElementById('status').innerHTML = 'OpenCV.js is ready.';
+      cv["onRuntimeInitialized"]=()=> {
+       document.getElementById("runSampl").disabled=false;
+      }
+    }
+    </script>
+    
+    <script async src = "https://docs.opencv.org/master/opencv.js" onload = "onOpenCvReady();" type = "text/javascript">
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md]: Please remove extra whitespaces here and above:
```html
     <script async src = "https://docs.opencv.org/master/opencv.js" onload = "onOpenCvReady();" type = "text/javascript">
```
```html
     <script async src="https://docs.opencv.org/master/opencv.js" onload="onOpenCvReady();" type="text/javascript">
```

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+ <h2>OpenCV.js</h2>
+  <p id = "status">OpenCV.js is loading...</p>
+          <img id = "imageSrc" src="http://bit.ly/fcc-relaxing-cat" />
+          <div> 
+          <input type = "button" id = "runSampl" onclick = "draw()" value="Run test" disabled=true />
+          </div>
+          <div>
+           <canvas id = "canvasOutput" >
+           </canvas>
+         </div>
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md]: Please format it in the following way (just indentations):
```html
 <h2>OpenCV.js</h2>
<p id = "status">OpenCV.js is loading...</p>
<img id = "imageSrc" src="http://bit.ly/fcc-relaxing-cat" />
<div> 
  <input type = "button" id = "runSampl" onclick = "draw()" value="Run test" disabled=true />
</div>
<div>
  <canvas id = "canvasOutput" ></canvas>
</div>
```

BTW, do we really need `<div>` tags?

Comment (SimpleVlad) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md]: We need tags for good visualization

Comment (SimpleVlad) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md]: But we can do differently

Comment (dkurt) [reply curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md]: Ok, let's keep it as is. Just fix indentation.

File: curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md [hunk 6]
```diff
@@ -45,23 +45,20 @@ tests:
...
+ 
+
+ <script type="text/javascript">
+    
     function draw() {
-      let mat = cv.imread("imageSrc");
-      let dst = new cv.Mat();  
-  
+      let mat=cv.imread("imageSrc");
+      let dst=new cv.Mat();
```
Comment (dkurt) [inline curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/openCV.md]:
```js
      let mat = cv.imread("imageSrc");
      let dst = new cv.Mat();  
````
But 
```html
<script async src="https://docs.opencv.org/master/opencv.js" onload="onOpenCvReady();" type="text/javascript">
```
So only for HTML attributes.

**Review by dkurt — APPROVED** [review]
LGTM

Next step is to enable test: we need to call cvtColor and compare it with user's `dst`

**Pull request merged**